docs/reference/pipeline-yaml-reference.md

```diff
@@ -1564,7 +1564,8 @@ after_pipeline:
     jobs:
       - name: Submit Metrics
         commands:
-          - echo "ci.duration:$SEMAPHORE_PIPELINE_TOTAL_DURATION|ms" | nc -w 3 -u statsd.example.com
+          - "export DURATION_IN_MS=$((SEMAPHORE_PIPELINE_TOTAL_DURATION * 1000))"
```

Comment:
<img width="536" alt="Screenshot 2021-09-29 at 21 34 04" src="https://user-images.githubusercontent.com/1779493/135336442-5125bff9-4737-4991-9588-5c11df31f1b7.png">

I wasn't quite sure about the syntax. Tested it here ^